oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/banyandb/client/request/TraceSearchRequest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.banyandb.client.request;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Singular;
+import lombok.experimental.SuperBuilder;
+import org.apache.skywalking.banyandb.Query;
+
+import java.util.List;
+
+@SuperBuilder
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class TraceSearchRequest extends HasMetadata {
+    // timeRange
+    private final TimeRange timeRange;
+
+    // query parameters
+    @Singular
+    private final List<TraceSearchQuery> queries;
+
+    // paging parameters: limit & offset
+    private final int limit;
+    private final int offset;
+
+    // query order
+    private final String queryOrderField;
+    private final SortOrder queryOrderSort;
```

Comment:
These should be exposed as `OrderBy` object with 2 fields as constructor.

Reply:
Done